src/PowerModelsRestoration.jl

```diff
@@ -4,9 +4,11 @@ import JuMP
 import InfrastructureModels
 import PowerModels
 import Memento
+import MathOptInterface
 
 const _IM = InfrastructureModels
 const _PM = PowerModels
+const _MOI = MathOptInterface
```

Comment:
PowerModels exports these contestants, so you should be able to do `_PM.OPTIMAL`